docs/SUMMARY.md

```diff
@@ -13,6 +13,7 @@
 * [Document Previews](Catalog/Preview.md)
 * [Embeddable iFrames](Catalog/Embed.md)
 * [Search & Query](Catalog/SearchQuery.md)
+* [Qurator Omni](advanced-features/qurator.md) AI Assistant
```

Comment:
Shouldn't this go under the advanced features section? Otherwise let's take the doc out of the advanced feature folder.

Reply:
Good point. Fixed.